src/App.js

```diff
@@ -13,7 +13,7 @@ function App() {
   const [longestSerie, setLongestSerie] = useState("");
 
   const multiplyNumbers = () => {
-    
+    // Implementation goes here
```

Comment:
Might be a good idea to leave a comment here to make it both easier to search and to let the user know we're waiting for our implementation.